Add multiple CIs to the diff chart

jessie-ross opened the pull request:
<!-- Describe your changes in detail. -->
**This PR adds multiple CIs to the diff chart.**

See the screenshot in Slack.

Resolves #702
<!-- Remember to include context: Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->

## How has this been tested?

<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->
<!-- Delete any bullet points that don't apply and add more details if needed. -->

- Automated tests that cover added/changed functionality

aaronyan reviewed (approved):
LGTM! Congrats on adding 99 and 50 credible intervals to Abacus! 🎉

aaronyan commented:
I've updated the Analyze Results FG page to use the multiple CIs and added a short section on it (PCYsg-Fqg-p2#experiment-results-graphs)!

jessie-ross commented:
> I've updated the Analyze Results FG page to use the multiple CIs and added a short section on it ([PCYsg-Fqg-p2#experiment-results-graphs](https://href.li/?PCYsg-Fqg-p2#experiment-results-graphs))!

Nice!
